python/lsst/summit/utils/butlerUtils.py

```diff
@@ -411,19 +411,19 @@ def updateDataIdOrDataCord(dataId: dict, **updateKwargs: Any) -> dict:
     return newId
 
 
-def fillDataId(butler: dafButler.Butler, dataId: dict) -> dict:
+def fillDataId(butler: dafButler.direct_butler.DirectButler, dataId: dafButler.DataId) -> Mapping[str, Any]:
```

Comment:
I'm surprised that the `DirectButler` part is necessary to be honest. The `dataId` also need to be able to take a dict though, that's the main thing.